src/actions.jl

```diff
@@ -2,132 +2,123 @@ abstract type Action end
 abstract type ByPermutations <: Action end
 abstract type ByLinearTransformation <: Action end
 
-abstract type InducedActionHomomorphism{A, T} end
+abstract type InducedActionHomomorphism{A,T} end
+#=
+implements:
+* features(action_hom)
+* action(hom::InducedActionHomomorphism) → Action
+* action(hom::InducedActionHomomorphism, g::GroupElement, feature) → the action of `g` on `feature`
+=#
 
 function induce(ac::Action, hom::InducedActionHomomorphism, g::GroupElement)
     throw(
         "No fallback is provided for $(typeof(ac)). You need to implement `induce(::$(typeof(ac)), ::$(typeof(hom)), ::$(typeof(g)))`.",
     )
 end
 
+induce(hom::InducedActionHomomorphism, g::GroupElement) = induce(action(hom), hom, g)
+
 decompose(x::Any, hom::InducedActionHomomorphism) = throw(
     "No fallback is provided for $(typeof(x)). You need to implement `decompose(::$(typeof(x)), ::$(typeof(hom)))`.",
 )
 
-coeff_type(ac::ByLinearTransformation) = throw(
-    "No fallback is provided for $(typeof(ac)). You need to implement `coeff_type(::$(typeof(ac)))`.",
+_coeff_type(ac::ByLinearTransformation) = throw(
```

Comment:
This is part of the interface and is implemented in SumOfSquares examples so it should not start with an `_`

Reply:
yeah, I forgot this one is part of the interface ;/ I'm not very happy about this, but I'm going to reverse the change ;)